service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/implementation/DigitalTwinsImpl.java

```diff
@@ -131,15 +126,12 @@ public Object call(ServiceResponseWithHeaders<Object, DigitalTwinGetDigitalTwinH
         }
         final String apiVersion = "2020-09-30";
         return service.getDigitalTwin(id, apiVersion)
-            .flatMap(new Func1<Response<ResponseBody>, Observable<ServiceResponseWithHeaders<Object, DigitalTwinGetDigitalTwinHeaders>>>() {
```

Comment:
Other folks in the team who are more familiar with this code can confirm, but isn't this generated code? Can we change it, or do we change it elsewhere and re-generate?

Reply:
Is there a way to disable analyzing generated code for warnings?

Reply:
Yeah, we wouldn't want to modify generated code. If there isn't a way to disable analyzing for specific file paths, we could add a suppression at class level.

Reply:
Agreed, reverted.